docs/KNOWN_BUGS

```diff
@@ -103,7 +103,6 @@ problems may have been fixed or changed somewhat since this was written.
  16.1 aws-sigv4 does not sign requests with * correctly
  16.2 aws-sigv4 does not sign requests with valueless queries correctly
  16.3 aws-sigv4 is missing the amz-content-sha256 header
```

Comment:
This one should be removed too, as it was fix here: https://github.com/curl/curl/pull/9995/files
note that sigv4 require amz-content-sha256 only with s3 (and maybe a few other API), but not always, so adding it all the time would have been a mistake.

Reply:
Also, we could add a line:
`aws-sigv4 does not canonize URL path`

Reply:
I have removed 16.3, but haven't added the new one (they seem to need refs to issues etc, maybe you have one?)

Reply:
I think the doc say that path needs to be canon: https://docs.aws.amazon.com/IAM/latest/UserGuide/create-signed-request.html
but I guess, that what happen here: https://github.com/curl/curl/issues/7559 just that the problem doesn't end with '*'.
Also, sorry if my comments are not pertinent, I'm slightly sick right now.

Reply:
Hope you feel better soon! I'm sure its a valid issue, but as I haven't looked into it I would rather not add the `KNOWN_ISSUE` in this MR (where commit history etc might cause someone to ask me about it 😀)